menu.json

```
@@ -6,23 +6,23 @@
   "featureToggle": null,
   "submenu": [{
     "key": "homepage",
-    "uriPath": "examples-starter/homepage",
+    "uriPath": "examples-starter/ShoppingLists",
```

URI paths usually are lowercase and separated by dashes.

In this case it would be

```suggestion
    "uriPath": "examples-starter/shopping-lists",
```

Furthermore, you might also want to change the entry point uri path to something other than `examples-starter`.